edx_proctoring/api.py

```diff
@@ -202,7 +203,7 @@ def get_review_policy_by_exam_id(exam_id):
     return ProctoredExamReviewPolicySerializer(exam_review_policy).data
 
 
-def update_exam(exam_id, exam_name=None, time_limit_mins=None, due_date=constants.MINIMUM_TIME,
+def update_exam(exam_id, exam_name=None, section_name=None, time_limit_mins=None, due_date=constants.MINIMUM_TIME,
```

Comment:
is it safe to add new variable in the middle of the existing list of variables? The same as create_exam function.